### ISSUE TITLE
sam validate command fails with EventBridgeRule

### ISSUE THREAD

**Issue by gavincornwell** [awslabs/aws-sam-cli#1781]
### Description

Calling `sam validate` on a template that contains an EventBridgeRule definition fails.

### Steps to reproduce

Execute `sam validate` using the following SAM template:

```
AWSTemplateFormatVersion: "2010-09-09"
Transform: AWS::Serverless-2016-10-31
Description: EventBridge test

Resources:

  EventBus:
    Type: AWS::Events::EventBus
    Properties:
      Name: MyEventBus

  EventAssignTrialLambda:
    Type: AWS::Serverless::Function
    Properties:
      MemorySize: 128
      Timeout: 10
      Runtime: nodejs12.x
      InlineCode: |
        exports.handler = async (event) => {
          console.log("event: " + JSON.stringify(event, null, 2));
          return "done";
        }
      Handler: events.lambda_handler
      Description: Processes the event
      Events:
        MyEvent:
          Type: EventBridgeRule
          Properties:
            EventBusName: MyEventBus
            Pattern:
              source:
                - MyEvent
```

### Observed result

The following error is returned:

```
Template provided at '<dir>/eventbridge.yaml' was invalid SAM Template.
Error: [InvalidResourceException('EventAssignTrialLambda', 'Event with id [MyEvent] is invalid. Resource dict has missing or invalid value for key Type. Event Type is: EventBridgeRule.')] ('EventAssignTrialLambda', 'Event with id [MyEvent] is invalid. Resource dict has missing or invalid value for key Type. Event Type is: EventBridgeRule.')
```

### Expected result

Validation should pass as this is a valid template that deploys successfully.

I guess the workaround for now is to either use the latest version of `cfn-lint` or define the EventBridge rule resource separately.

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

1. OS: MacOS 10.14.6
2. SAM 0.40.0
3. cfn-lint 0.27.4

**Comment by gavincornwell** [comment]
It looks like this has been resolved in the latest release (0.41.0).

@sriram-mv, shall I close the issue?

**Comment by sriram-mv** [comment]
Perfect, thanks for responding, will close this issue.
